src/components/lyne-toast/lyne-toast.custom.d.ts

```diff
@@ -0,0 +1,75 @@
+import { AnimationBuilder } from '../../global/core/components/animations/animation-interface';
+
+export interface InterfaceLyneToastAttributes {
+  verticalPosition: 'top' | 'bottom';
+  horizontalPosition: 'left' | 'center' | 'right' | 'start' | 'end';
+  politeness: 'off' | 'assertive' | 'polite';
```

Comment:
in lyne-alert I named it "ariaLivePoliteness" to explicitly mention the aria context. what are your thoughts?